DuckDuckGo/Suggestion.swift

```diff
@@ -18,9 +18,22 @@
 //
 
 import Foundation
+import Core
 
-struct Suggestion {
+struct Suggestion: Decodable {
 
     let type: String
     let suggestion: String
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.singleValueContainer()
+        let dictionary = try container.decode([String: String].self)
```

Comment:
Shouldn't we use `[String: String?]` to be on a safe side?

Reply:
On a second thought it is most likely fine - this is going to throw in case of such error anyway.